misc/scripts/gen_boarddefs.py

```diff
@@ -156,6 +156,10 @@ def exact_series(self) -> str:
     ChipInfo("CH32X035F7P6", 62, 20, 48, "TSSOP20"),
     ChipInfo("CH32X033F8P6", 62, 20, 48, "TSSOP20"),
     # CH32L10x
+    ChipInfo("CH32L103F8P6", 64, 20, 96, "LQFP48"),
+    ChipInfo("CH32L103F8U6", 64, 20, 96, "LQFP48"),
+    ChipInfo("CH32L103G8R6", 64, 20, 96, "LQFP48"),
+    ChipInfo("CH32L103K8U6", 64, 20, 96, "LQFP48"),
```

Comment:
FYI, The `96, "LQFP48"` in each line is the number of pins and package, so that should all be
```py
    ChipInfo("CH32L103F8P6", 64, 20, 20, "TSSOP20"),
    ChipInfo("CH32L103F8U6", 64, 20, 20, "QFN20"),
    ChipInfo("CH32L103G8R6", 64, 20, 28, "QSOP28"),
    ChipInfo("CH32L103K8U6", 64, 20, 32, "QFN32"),
```
According to https://www.wch-ic.com/downloads/CH32L103DS0_PDF.html

Reply:
Oh, sorry. But 96 means CPU frequency according to line 12

Reply:
True.